yarn.lock

```diff
@@ -24713,6 +24706,18 @@ workspace-tools@0.27.0, workspace-tools@^0.27.0:
     js-yaml "^4.1.0"
     micromatch "^4.0.0"
 
+workspace-tools@^0.29.0:
```

Comment:
can you pls also bump this package in monorepo root so we wont get any dups? ty

<img width="1312" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/8063a888-29a3-42db-88d3-20f4a512683b">

Reply:
I bumped the version, but there are still dupes since there is an inner dependency that still depends on version `0.27.0`.

Reply:
yeah beachball. I'll deal with it later. thanks !